src/System.Reflection/tests/AssemblyNameTests.cs

```diff
@@ -301,11 +301,10 @@ public void FullName(string name, string expectedName)
         {
             AssemblyName assemblyName = new AssemblyName(name);
 
-            expectedName = expectedName.ToLowerInvariant();
-            string extended = $"{expectedName}, Culture=neutral, PublicKeyToken=null".ToLowerInvariant();
-            string afn = assemblyName.FullName.ToLowerInvariant();
+            expectedName = expectedName;
```

Comment:
This line is a nop.  Just delete it?

Reply:
Oh yeah, sorry... that's what I get for just blindly running a find and replace on the snippet.